Add stgUSD and tgUSD jetton configurations

alan890104 opened the pull request:
Hello, I'm founder of Torch Finance. We recently launched tgUSD, a cross-chain yield-bearing stablecoin built on TON.

As a loyal Tonkeeper user myself, I sincerely hope that users will be able to see tgUSD and its staked version stgUSD properly displayed in the wallet — including price data and verified badges. Both tokens have already been verified on Tonviewer, TONScan, and DeDust.

We would greatly appreciate your support in helping us get listed and recognized within Tonkeeper. Thank you!

Here are some supporting media materials that you can use as reference for the review.
1. [TON Foundation Post](https://x.com/TorchTon/status/1920086294346297695)
2. [Foresight News](https://t.me/foresightnews/75113)

alan890104 committed:
Add stgUSD and tgUSD jetton configurations

salateur commented:
Hello.
Syntax error in the filetype: you must use .yaml instead of yml.
Also, I'd suggest change/delete image part: it could be broken, not sure if ipfs direct link is supported.
Either upload it for direct access (example.io/example/example.png) or use the current metadata hence delete the "image" line, thanks. (but maybe it will work out and you wont need do any changes... :D)

alan890104 committed:
Add configurations for stgUSD and tgUSD jettons

alan890104 commented:
Sorry for the trouble, and thank you for the reminder.
I have updated the file extension to **.yaml** and removed the **image** field, so it will now read from the jetton metadata.
I’d appreciate it if you could help me proceed with the merge. Thanks a lot!
